### ISSUE TITLE
provide noVerify input param

### ISSUE THREAD

**Issue by AnWeber** [HaaLeo/publish-vscode-extension#8]
I am using this great github action to upload an extension for both openvsx and vsce. In my extension I need the setting enableProposedApi: true because I use the Noteboook Api. However, when publishing, this setting is blocked by vsce and [noVerify](https://github.com/microsoft/vscode-vsce/blob/b0c820449b7df5e34d0ebc5bd80cb3a6ecc90ea0/src/publish.ts) must be explicitly enabled.
According to [vsce#469](https://github.com/microsoft/vscode-vsce/issues/469) this setting is also the suggested solution

The best solution would probably be to be able to directly specify all possible parameters for both ovse and vsce.

```
- name: Publish to Visual Studio Marketplace
      uses: HaaLeo/publish-vscode-extension@v0
      with:
        pat: ${{ secrets.VSCE_TOKEN }}
        registryUrl: https://marketplace.visualstudio.com
        extensionFile: ${{ steps.publishToOpenVSX.outputs.vsixPath }}
        vsceOptions: 
            noVerify: true
        openvsxOptions: 
            ...
```

But if I read the api of @action/core correctly, only string is supported here and implementing the solution seems rather complex.

**Comment by HaaLeo** [comment]
Hi @AnWeber thx for raising this feature request. I also searched the docs and it seems it is only possible to pass string values to the `actions/core` API.
I think there are two main options here:
1. Passing in the vsce options as stringified JSON and parse them during runtime. This would allow us to clearly seperate between openvsx and vsce specific options. However, imho I think passing a JSON datastructure could be quite error prone and is quite unintuitive.
     ```yml
        - name: Publish to Visual Studio Marketplace
          uses: HaaLeo/publish-vscode-extension@v0
          with:
              pat: ${{ secrets.VSCE_TOKEN }}
              registryUrl: https://marketplace.visualstudio.com
              extensionFile: ${{ steps.publishToOpenVSX.outputs.vsixPath }}
              vsceOptions:  '{"noVerify": true}'
              openvsxOptions: '{}'
     ``` 
1. Introduce `noVerify` as top level option. Then vsce and vsx options will be mixed up at top level. I think this approach fits better into the specification of GitHub action inputs. And maybe we could clearly name it as `vsceNoVerify` or similar.
     ```yml
        - name: Publish to Visual Studio Marketplace
          uses: HaaLeo/publish-vscode-extension@v0
          with:
              pat: ${{ secrets.VSCE_TOKEN }}
              registryUrl: https://marketplace.visualstudio.com
              extensionFile: ${{ steps.publishToOpenVSX.outputs.vsixPath }}
              noVerify: true
     ```

**Comment by AnWeber** [comment]
I also thought of variant 1, but I don't find it a nice api. I would implement variant 2, but extends ActionOptions

```javascript
import { IPublishVSIXOptions as VSCEPublishOptions } from 'vsce';
import { PublishOptions as OVSXPublishOptions } from 'ovsx';

export interface ActionOptions extends OVSXPublishOptions, VSCEPublishOptions   {
    dryRun: boolean;
}
```

**Comment by HaaLeo** [comment]
The `ActionOptions` interface is meant to represent _only_ the inputs of this GitHub action. When I originally implemented this action I thought it would be good practice to align the action's inputs with the ovsx CLI.
When we now extend the interface as you suggested, the interface suggests that all those options are valid inputs to the action, although they may are not. Therefore, I would like to keep the interface _as restrictive as possible_ and only indicate inputs which are truly supported by the action. 
So maybe something like this:
```typescript
export interface ActionOptions extends OVSXPublishOptions   {
    dryRun: boolean;
    noVerify: boolean;
}
```
 
or even completely decouple the interface from the ovsx CLI options: 
```typescript
export interface ActionOptions   {
    dryRun: boolean;
    extensionFile: string; // ovsx's extensionFile option for backwards compability
    // ... many more
    noVerify: boolean;
    web: boolean;  // Must be added to the acion.yml file as well.
}
```

**Comment by AnWeber** [comment]
_"When we now extend the interface as you suggested, the interface suggests that all those options are valid inputs to the action"_
yes, that was intentional:-) But if it should be restrictive, I would decouple the interface from OVSXPublishOptions and explicitly name all options. Both sound good for me.

**Comment by HaaLeo** [comment]
Sounds good to me :). I'd love to review a PR for this issue, including some unit tests.

**Comment by HaaLeo** [comment]
Released with [v0.4.0](https://github.com/HaaLeo/publish-vscode-extension/releases/tag/v0.4.0)

**Comment by AnWeber** [comment]
thanks. did work

```
Run HaaLeo/publish-vscode-extension@v0
  with:
    pat: ***
    registryUrl: https://marketplace.visualstudio.com
    extensionFile: httpbook-0.10.0.vsix
    noVerify: true
    yarn: false
    dryRun: false
Specifically, running GitHub Action HaaLeo/publish-vscode-extension@v0.3.1.
Package the Extension
  The extension was already packaged. Skip packaging.

Publish the Extension
  (node:1768) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
  Publishing anweber.httpbook@0.10.0...
   INFO  Extension URL (might take a few minutes): https://marketplace.visualstudio.com/items?itemName=anweber.httpbook
   INFO  Hub URL: https://marketplace.visualstudio.com/manage/publishers/anweber/extensions/httpbook/hub
   DONE  Published anweber.httpbook@0.10.0.
